keyboards/handwired/dactyl_rah/readme.md

```diff
@@ -0,0 +1,78 @@
+# Dactyl with Arduino Pro Micro by github.com/ralphie02
```

Comment:
Most of this could be removed and replaced with the default template.

Reply:
Replaced readme with ref to the dactyl_micro readme